.github/workflows/lint.yml

```diff
@@ -0,0 +1,44 @@
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Node 16
+        uses: actions/setup-node@v2
+        with:
+          cache: yarn
+          node-version: 16
+      - name: Dependencies
+        run: yarn --frozen-lockfile --ignore-scripts
+      - name: ESLint
+        run: yarn run lint
```

Comment:
We don't have ESLint, maybe we should just remove this and then add it if we decide to add eslint?